src/evops/utils/MetricsUtils.py

```diff
@@ -73,42 +88,38 @@ def __get_tp(
     """
     true_positive = 0
 
-    unique_gt_labels = np.unique(gt_labels)
-    unique_gt_labels = np.delete(
-        unique_gt_labels,
-        np.where(unique_gt_labels == evops.metrics.constants.UNSEGMENTED_LABEL)[0],
-    )
-    assert (
-        unique_gt_labels.size != 0
-    ), "Incorrect ground truth labels unique count, most likely no labels other than UNSEGMENTED_LABEL"
-
-    unique_pred_labels = np.unique(pred_labels)
-    unique_pred_labels = np.delete(
-        unique_pred_labels,
-        np.where(unique_pred_labels == evops.metrics.constants.UNSEGMENTED_LABEL)[0],
-    )
-    assert (
-        unique_pred_labels.size != 0
-    ), "Incorrect predicted labels unique count, most likely no labels other than UNSEGMENTED_LABEL"
+    unique_gt_labels = __filter_unsegmented_unique(gt_labels)
+    unique_pred_labels = __filter_unsegmented_unique(pred_labels)
```

Comment:
do we really need unique in func? is it possible to use only `__filter_unsegmented()` and find unique here before this call?